language/language_samples.rb

```diff
@@ -36,9 +38,11 @@ def sentiment_from_cloud_storage_file project_id:, storage_path:
 
   language  = Google::Cloud::Language.new project: project_id
   document  = language.document storage_path
-  sentiment = document.sentiment
-
-  puts "#{sentiment.score} (#{sentiment.magnitude})"
+  puts "Overall document sentiment: (#{document.sentiment.score})"
+  puts "Sentence level sentiment:"
+  document.sentiment.sentences.each do |sentence|
+    puts "#{sentence.text}: (#{sentence.sentiment.score})"
```

Comment:
Tests should be added to verify multiple sentences were identified by the sample.